### PR TITLE
Remove withdrawal Queue

### PR DESCRIPTION
**Pull request by potuz**
This PR implements the beacon state for Capella as per https://github.com/ethereum/consensus-specs/pull/3068

### CODE REVIEW

File: proto/prysm/v1alpha1/attestation/aggregation/attestations/attestations.go [hunk 1]
```diff
@@ -27,11 +27,11 @@ var ErrInvalidAttestationCount = errors.New("invalid number of attestations")
 // Aggregation occurs in-place i.e. contents of input array will be modified. Should you need to
 // preserve input attestations, clone them before aggregating:
 //
-//   clonedAtts := make([]*ethpb.Attestation, len(atts))
-//   for i, a := range atts {
-//       clonedAtts[i] = stateTrie.CopyAttestation(a)
-//   }
-//   aggregatedAtts, err := attaggregation.Aggregate(clonedAtts)
+//	clonedAtts := make([]*ethpb.Attestation, len(atts))
```
Comment (terencechain) [inline proto/prysm/v1alpha1/attestation/aggregation/attestations/attestations.go:30]: why does your PR always do this?

File: beacon-chain/state/state-native/beacon_state_mainnet.go [hunk 2]
```diff
@@ -47,9 +47,8 @@ type BeaconState struct {
 	nextSyncCommittee                   *ethpb.SyncCommittee                    `ssz-gen:"true"`
 	latestExecutionPayloadHeader        *enginev1.ExecutionPayloadHeader        `ssz-gen:"true"`
 	latestExecutionPayloadHeaderCapella *enginev1.ExecutionPayloadHeaderCapella `ssz-gen:"true"`
-	withdrawalQueue                     []*enginev1.Withdrawal                  `ssz-gen:"true" ssz-max:"1099511627776"`
 	nextWithdrawalIndex                 uint64                                  `ssz-gen:"true"`
-	nextPartialWithdrawalValidatorIndex eth2types.ValidatorIndex                `ssz-gen:"true"`
+	lastWithdrawalValidatorIndex        eth2types.ValidatorIndex                `ssz-gen:"true"`
```
Comment (terencechain) [inline beacon-chain/state/state-native/beacon_state_mainnet.go:51]: I think we can remove ``ssz-gen:"true"``, it's probably better to do it in another PR

**Review by rauljordan — APPROVED** [review]
Very easy to review, thanks @potuz

File: proto/prysm/v1alpha1/attestation/aggregation/attestations/attestations.go (hunk 1 above)
Comment (potuz) [reply proto/prysm/v1alpha1/attestation/aggregation/attestations/attestations.go:30]: have no clue, I just run update-go-pbs

Comment (terencechain) [reply proto/prysm/v1alpha1/attestation/aggregation/attestations/attestations.go:30]: That's weird. Our `./hack/update-go-pbs.sh` don't do this. Hopefully we don't mess the next person up
